src/main/java/seedu/duke/task/Item.java

```diff
@@ -0,0 +1,184 @@
+package seedu.duke.task;
+
+import seedu.duke.DukeException;
+import seedu.duke.common.Messages;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+// Renamed from previous Task.java with some modifications.
+/**
+ * Represents an item in the list.
+ */
+public class Item implements Comparable<Item> {
+    public static DateTimeFormatter DATETIME_PARSE_FORMAT = DateTimeFormatter.ofPattern("dd-MM-yyyy");
+    public static DateTimeFormatter DATETIME_PRINT_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy");
+
+    protected String description;
+    protected boolean isDone;
+    protected int priority;
+    protected String category;
+    protected LocalDate date;
```

Comment:
I was wondering if these fields should be part of `Item` or they should be part of `Task` because not all items we're adding might have these fields? For example if new classes like `Expenses` extends `Item` then it may not need all these fields, or let the fields be set.